src/window_main/components/economy/EconomyRow.tsx

```diff
@@ -171,16 +174,29 @@ function countDupesArray(array: string[]): Record<string, number> {
 
 function CardPoolAddedEconomyValueRecord(props: CardPoolAddedEconomyValueRecordProps) {
   const { addedCardIds, aetherizedCardIds } = props;
-  const addedUniques = (addedCardIds.length > 0) ? countDupesArray(addedCardIds) : undefined;
-  const aetherUniques = (aetherizedCardIds.length > 0) ? countDupesArray(aetherizedCardIds) : undefined;
+
+  return (<>{AddedCardList(addedCardIds)}{AetherizedCardList(aetherizedCardIds)}</>);
```

Comment:
Then you can re-write this as
```suggestion
  return (
    <>
      <InventoryCardList cardList={addedCardIds} />
      <InventoryCardList cardList={aetherizedCardIds} isAetherized />
    </>
  );
```